src/memo/memo.ml

```diff
@@ -781,6 +789,16 @@ module Async = struct
   type nonrec ('i, 'o) t = ('i, 'o, 'i -> 'o Fiber.t) t
 end
 
+let current_run =
+  let f () = Run.current () in
+  let memo =
+    create "current-run" ~doc:"current run"
+      ~input:(module Unit)
+      ~output:(Allow_cutoff (module Run))
```

Comment:
We know for sure that the result will be different at every run, so the cutoff optimisation will never trigger. We should use `Simple` here